plumpy/events.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 """Event and loop related classes and functions"""
 import sys
+
 from tornado import ioloop
```

Comment:
`import asyncio`

and then 

```
new_event_loop = asyncio.new_event_loop
set_event_loop = asyncio.set_event_loop
def run_until_complete(fut, loop=None):
    loop = loop or asyncio.get_event_loop()
    return loop.run_until_complete(fut)
```

(note I haven't tried this code)

Reply:
Yes, in fact, this file is deleted in my implement. I should make a clean up before I ask for review, sorry about that.